cvTargetAssist.py

```diff
@@ -18,556 +43,860 @@
 from depthai_nodes.node import ParsingNeuralNetwork, GatherData
 from depthai_nodes.message.keypoints import Keypoint
 
-"""
-In the default YOLO11 pose model, there are 17 keypoints,
-each representing a different part of the human body.
-Here is the mapping of each index to its respective body joint:
-
-0    Nose
-1    Left Eye
-2    Right Eye
-3    Left Ear
-4    Right Ear
-5    Left Shoulder
-6    Right Shoulder
-7    Left Elbow
-8    Right Elbow
-9    Left Wrist
-10   Right Wrist
-11   Left Hip
-12   Right Hip
-13   Left Knee
-14   Right Knee
-15   Left Ankle
-16   Right Ankle
-"""
-
-class SETTINGS:
-    enable_visualizer = False
-    SERIAL_OUTPUT = True
-    log_level = dai.LogLevel.WARN
-    FPS = 10
-    CLOSE_MATCH = 0.16
-    MEDIUM_MATCH = 0.22
-    FAR_MATCH = 0.30
-    MODE = 'POSE'   # 'POSE' or 'FACE'
-    STATE = 'TEST'
-
-
+# ======================================================================================
+# --- Configuration ---
+# ======================================================================================
+
+class ScriptSettings:
```

Comment:
Implement an argparse function that will set all these parameters.  It should default to 'POSE' for detection mode and False for serial output and visualizer.  Otherwise the values present are sufficient for defaults.

Reply:
Also make it support a short flag version for the visualizer.  `-v` seems good.